### ISSUE TITLE
Create indexDeep

### ISSUE THREAD

**Issue by j-harbin** [ArgoCanada/argoFloats#188]
The following is a checklist to create a deep Argo index named `indexDeep`.
* [x] Decide on the area of interest
* [x] Create create_indexDeep in argoFloats/create_data (use create_index as guide)
* [x] Update Makefile in create_data by adding
    ```
    indexDeep:force
    <TAB>Rscript create_indexBgc.R > create_indexBgc.out
     ```
* [x] Update Makefile in create_data, by adding
    ```
    install_indexDeep:
    <TAB>cp indexDeep.rda ../data
    ```
* [x] Update `AllClass.R` to include` indexDeep`

**Comment by dankelley** [comment]
@j-harbin I like your checklist.  I think you ought to copy it into a file in this dir, named `README.md`.  You might want to add some lines to explain some more details.  And, the way these things usually work is that you see new steps, so you'll want to look for that.

Please note that I edited your previous comment to make it proper markdown format.  You can check the format on github.com by looking in the directory ... a `README.md` file gets formatted in such a page.

**Comment by j-harbin** [comment]
Hi @dankelley ,

I know you're in a meeting right now, so no rush on this, but I've completed the checklist in branch issue188, commit addda83bb06e60f428861f4b1f9ea81d2137ca04 . When I 'Check" the package, however, I receive the error;

```R
  > ### Title: A sample index of deep Argo
  > ### Aliases: indexDeep
  > 
  > ### ** Examples
  > 
  > library(argoFloats)
  > data(indexDeep)
  Warning in data(indexDeep) : data set ‘indexDeep’ not found
  > plot(indexDeep, bathymetry=FALSE)
  Error in plot(indexDeep, bathymetry = FALSE) : 
    object 'indexDeep' not found
  Execution halted

1 error x | 0 warnings ✓ | 0 notes ✓
Error: R CMD check found ERRORs
Execution halted

Exited with status 1.
```

I believe this is an issue with needing to make the Makefile. When I try that in the terminal, however, I receive:
```R
This Makefile is disabled, to avoid altering existing datasets. Experts know what to do...
```

So I believe I need an experts help!

**Comment by dankelley** [comment]
Let's ms/teams.  Would 10:30 be ok?  or 11:00?

**Comment by j-harbin** [comment]
11:00 works great.

**Comment by j-harbin** [comment]
This issue has been solved in commit 4a913651f2db702d3f5c1d3d5ddd7082f7085446 of the develop branch.
